### DISCUSSION TITLE
Cannot Pull Custom Package

### DISCUSSION THREAD

**Discussion by pczinser** [nanovms/ops#1352] [Q&A] [answered]
I have made a custom package (https://repo.ops.city/v2/packages/pczinser/named/show), but I cannot run an `ops pkg load pczinser/named:9.18.6` on my desktop. 

I get the following error: 
`can not download file from: https://repo.ops.city/v2/packages/pczinser/named/9.18.6.tar.gz, status: 404 Not Found`

If you look at the package on repo.ops.city it displays the contents of the README that I build into the package .  I can also find the package if I run an `ops pkg search named`. 

I create this package in a GitHub Actions pipeline: https://github.com/pczinser/nanovms-bind
as the last step it runs `$HOME/.ops/bin/ops pkg login ${{ secrets.OPS_REPO_KEY }} && $HOME/.ops/bin/ops pkg push named_9.18.6` Is there something else I need to be doing when I upload it?

**Comment by eyberg** [comment, marked as answer]
sorry about that, we found an issue that was preventing uploads; if you delete that pkg and try again let me know if that works; keep in mind the name/version are unique right now

**Reply by pczinser** [reply]
I was able to delete it and re upload it and I am able to pull it down with `ops pkg load ...`. Thanks.
